.pipelines/stages/jobs/capi-packaging-job.yml

```diff
@@ -0,0 +1,136 @@
+parameters:
+- name: arch
+  type: string
+- name: ep
+  type: string
+- name: ort_version
+  type: string
+- name: cuda_version
+  type: string
+  default: ''
+- name: os
+  type: string
+  values:
+  - 'linux'
+  - 'win'
+- name: build_config
+  type: string
+  default: 'release'
+
+jobs:
+- job: capi_${{ parameters.os }}_${{ parameters.ep }}_${{ parameters.arch }}_packaging
+  ${{ if eq(parameters.os, 'linux') }}:
+    ${{ if eq(parameters.arch, 'x64') }}:
+      pool: 'onnxruntime-Ubuntu2204-AMD-CPU'
+    ${{ elseif eq(parameters.arch, 'arm64') }}:
+      pool: 'onnxruntime-genai-Ubuntu2004-ARM-CPU'
+    ${{ else }}:
+      pool: 'nnxruntime-Ubuntu2204-AMD-CPU'
+  ${{ if eq(parameters.os, 'win') }}:
+    ${{ if eq(parameters.arch, 'x64') }}:
+      pool: 'onnxruntime-Win-CPU-2022'
+    ${{ elseif eq(parameters.arch, 'arm64') }}:
+      pool: 'onnxruntime-genai-windows-vs-2022-arm64'
+    ${{ else }}:
+      pool: 'onnxruntime-Win-CPU-2022'
+  timeoutInMinutes: 180
+  #  set variables here to be used in the template and steps
+  variables:
+  - name: arch
+    value: ${{ parameters.arch }}
+  - name: artifactName
+    value: 'onnxruntime-genai-${{ parameters.os }}-${{ parameters.ep }}-${{ parameters.arch }}'
+  - name: build_config
+    value: ${{ parameters.build_config }}
+  - name: buildDir
+    value: 'build/${{ parameters.ep }}'
+  - name: cuda_version
+    value: ${{ parameters.cuda_version }}
+  - name: ep
+    value: ${{ parameters.ep }}
+  - name: ort_version
+    value: ${{ parameters.ort_version }}
+  - name: GDN_CODESIGN_TARGETDIRECTORY
+    value: '$(Build.ArtifactStagingDirectory)/nuget'
+  - name: os
+    value: ${{ parameters.os }}
+  - name: feed_name
+    ${{ if and(eq(parameters.cuda_version, '12.2'), eq(parameters.ep, 'cuda')) }}:
+      value: 'ort-cuda-12-nightly'
+    ${{ else }}:
+      value: '7982ae20-ed19-4a35-a362-a96ac99897b7'
+  - name: ort_filename
+    ${{ if eq(parameters.ep, 'cpu') }}:
+      value: 'Microsoft.ML.OnnxRuntime'
+    ${{ elseif eq(parameters.ep, 'cuda') }}:
+      ${{if eq(parameters.os, 'win') }}:
+        value: 'Microsoft.ML.OnnxRuntime.Gpu.Windows'
+      ${{ else }}:
+        value: 'Microsoft.ML.OnnxRuntime.Gpu.Linux'
+    ${{ elseif eq(parameters.ep, 'directml')}}:
+      value: 'Microsoft.ML.OnnxRuntime.DirectML'
+    ${{ elseif eq(parameters.ep, 'rocm')}}:
+      value: 'Microsoft.ML.OnnxRuntime.ROCm'
+    ${{ else }}:
+      value: 'Microsoft.ML.OnnxRuntime'
+
+  - name: ortHome
+    value: 'ort'
+  - name: dml_dir
+    value: 'Microsoft.AI.DirectML.1.14.2'
```

Comment:
They just upgraded this version to 1.15.0

Reply:
done